<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="4" passed="1">
  <reporter-output>
    <line>
      <![CDATA[<a href='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyHomePageHeader_25_02_2021_12_25_34.png'> <img src='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyHomePageHeader_25_02_2021_12_25_34.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyLoggedInUserTest_25_02_2021_12_25_39.png'> <img src='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyLoggedInUserTest_25_02_2021_12_25_39.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyAuthentication_25_02_2021_12_25_42.png'> <img src='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyAuthentication_25_02_2021_12_25_42.png' height='100' width='100'/> </a>]]>
    </line>
  </reporter-output>
  <suite name="Test Suite" duration-ms="15074" started-at="2021-02-25T17:25:27Z" finished-at="2021-02-25T17:25:42Z">
    <groups>
    </groups>
    <test name="Orange Test" duration-ms="15074" started-at="2021-02-25T17:25:27Z" finished-at="2021-02-25T17:25:42Z">
      <class name="com.qa.orangehrm.tests.HomePageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="setUp" is-config="true" duration-ms="3254" started-at="2021-02-25T17:25:27Z" finished-at="2021-02-25T17:25:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyHomePageTitle()[pri:1, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="verifyHomePageTitle" duration-ms="41" started-at="2021-02-25T17:25:31Z" description="verify home page title" finished-at="2021-02-25T17:25:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomePageTitle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="tearDown" is-config="true" duration-ms="418" started-at="2021-02-25T17:25:31Z" finished-at="2021-02-25T17:25:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="setUp" is-config="true" duration-ms="2640" started-at="2021-02-25T17:25:31Z" finished-at="2021-02-25T17:25:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifyHomePageHeader()[pri:2, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="verifyHomePageHeader" duration-ms="16" started-at="2021-02-25T17:25:34Z" description="verify home page header" finished-at="2021-02-25T17:25:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dashboard!] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dashboard!] but found [null]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.orangehrm.tests.HomePageTest.verifyHomePageHeader(HomePageTest.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyHomePageHeader_25_02_2021_12_25_34.png'> <img src='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyHomePageHeader_25_02_2021_12_25_34.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyHomePageHeader -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="tearDown" is-config="true" duration-ms="415" started-at="2021-02-25T17:25:34Z" finished-at="2021-02-25T17:25:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="setUp" is-config="true" duration-ms="4584" started-at="2021-02-25T17:25:34Z" finished-at="2021-02-25T17:25:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifyLoggedInUserTest()[pri:3, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="verifyLoggedInUserTest" duration-ms="9" started-at="2021-02-25T17:25:39Z" description="verify account name method" finished-at="2021-02-25T17:25:39Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.orangehrm.tests.HomePageTest.verifyLoggedInUserTest(HomePageTest.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[<a href='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyLoggedInUserTest_25_02_2021_12_25_39.png'> <img src='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyLoggedInUserTest_25_02_2021_12_25_39.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyLoggedInUserTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="tearDown" is-config="true" duration-ms="418" started-at="2021-02-25T17:25:39Z" finished-at="2021-02-25T17:25:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="setUp" is-config="true" duration-ms="2575" started-at="2021-02-25T17:25:39Z" finished-at="2021-02-25T17:25:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifyAuthentication()[pri:4, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="verifyAuthentication" duration-ms="6" started-at="2021-02-25T17:25:42Z" description="verify page URL" finished-at="2021-02-25T17:25:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.qa.orangehrm.tests.HomePageTest.verifyAuthentication(HomePageTest.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyAuthentication_25_02_2021_12_25_42.png'> <img src='/Users/ahmetcturk/git/OrangeHRM/Framework_/target/surefire-reports/failure_screenshots/verifyAuthentication_25_02_2021_12_25_42.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyAuthentication -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.orangehrm.tests.HomePageTest@7692d9cc]" name="tearDown" is-config="true" duration-ms="420" started-at="2021-02-25T17:25:42Z" finished-at="2021-02-25T17:25:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.orangehrm.tests.HomePageTest -->
    </test> <!-- Orange Test -->
  </suite> <!-- Test Suite -->
</testng-results>
